opencti-platform/opencti-front/src/utils/hooks/useScale.ts

```diff
@@ -1,19 +1,62 @@
+import { clone } from 'ramda';
 import useEntitySettings from './useEntitySettings';
 import { Scale, ScaleConfig, Tick } from '../../private/components/settings/sub_types/scaleConfiguration/scale';
 
-const defaultScale: ScaleConfig = {
+const admiraltyScale: ScaleConfig = {
   better_side: 'min',
   min: {
     value: 0,
     color: '#f44336',
-    label: 'Low',
+    label: '6 - Truth Cannot be judged',
   },
   max: {
     value: 100,
     color: '#6e44ad',
     label: 'Out of Range',
   },
   ticks: [
+    { value: 1, color: '#f57423', label: '5 - Improbable' },
```

Comment:
I am not really fan to have this in front-end
It's possible to declare it in back-end ?

Reply:
Well, for a first version of the confidence templates, we don't need to have them in the backend since you can't change them, it's only used to fill the form. 
Why don't you like it exactly ?

Reply:
I don't like the fact of having raw data on the front side, but as you say.. it's only used on the front side for the moment.
And we don't have any templating logic on the back side for this part.